js-project-guidelines.md

```diff
@@ -179,17 +183,24 @@ Inside the package.json, the main file exported is the one from the auto-generat
 
 Here you can find samples for [Travis](examples/travis.example.yml) and [circle](examples/circle.example.yml).
 
+We also use [coveralls.io](https://coveralls.io/) to automatically publish coverage reports. This is done from travis using
+
+```yml
+script:
+  - npm run coverage
+after_success:
+  - npm run coverage-publish
+```
+
 ##### `.gitignore`
 
-To avoid checking in the dist and lib folders, the `.gitignore` file should at least contain:
+To avoid checking in unwanted files, the `.gitignore` file should follow
+this [example](examples/.gitignore).
```

Comment:
The word should is a bit hard. This should be reworded to say 'A .gitignore is useful. An example is provided in examples'